src/Python/divik2spectre.py

```diff
@@ -0,0 +1,149 @@
+"""Convert MATLAB data file into Spectre txt format
+
+divik2spectre.py
+Converts MATLAB data file into Spectre txt format
+
+Copyright 2017 Spectre Team
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import argparse as agp
+import h5py
+import numpy as np
+import scipy.io as scio
+from tqdm import tqdm
+from typing import List
+
+
+def parse_args():
+    """Parses command line arguments, requiring source and destination paths"""
```

Comment:
If it's not a problem, please specify converter input format somewhere in the docs of this script or the project (or just provide a link to description).

Reply:
Added on the top of the script.